core/vm/operations_verkle.go

```diff
@@ -74,14 +74,15 @@ func gasExtCodeHash4762(evm *EVM, contract *Contract, stack *Stack, mem *Memory,
 
 func makeCallVariantGasEIP4762(oldCalculator gasFunc) gasFunc {
 	return func(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
+		address := stack.Back(1).Bytes20()
 		gas, err := oldCalculator(evm, contract, stack, mem, memorySize)
 		if err != nil {
 			return 0, err
 		}
-		if _, isPrecompile := evm.precompile(contract.Address()); isPrecompile {
```

Comment:
`contract` uses the current contract, i.e. the caller. I suppose this bug was introduced when I moved things from the `CALL` to this dedicated function. It was correct then, but the meaning of `contract` has changed.